waitlist/ts3/connection.py

```diff
@@ -60,7 +60,7 @@ def __do_keep_alive(self):
                     try:
                         self.conn.send_keepalive()
                     except Exception as ex:
-                        logger.debug('Sending keepalive to ts3 connection failed', ex)
+                        logger.debug('Sending keepalive to ts3 connection failed error: %s', ex)
```

Comment:
```suggestion
                        logger.debug('Sending keepalive to ts3 connection failed', exc_info=ex)
```
this should be the way to fix it, I would suggest.